load_balancer/nginx.conf

```diff
@@ -10,15 +10,32 @@ http {
     default_type  application/json;
 
     keepalive_timeout  65;
-    
-    upstream messenger_upstream {
-      server messenger:4000;
-    }
+
     server {
         listen       80;
-        server_name  localhost;
+
         location / {
-          proxy_pass http://messenger_upstream;
+          proxy_pass http://localhost:8080;
+        }
+        
+        location /health {
+          access_log off;
+          add_header 'Content-Type' 'text/plain';
+          return 200 "OK";
```

Comment:
**Annotation**:
Pretty basic health check endpoint.  We make sure these don't pollute the access log (which is a list of incoming requests).